static/app/views/replays/searchBar.tsx

```diff
@@ -0,0 +1,151 @@
+import {useEffect} from 'react';
+import assign from 'lodash/assign';
+import flatten from 'lodash/flatten';
+import memoize from 'lodash/memoize';
+import omit from 'lodash/omit';
+
+import {fetchTagValues} from 'sentry/actionCreators/tags';
+import SmartSearchBar from 'sentry/components/smartSearchBar';
+import {NEGATION_OPERATOR, SEARCH_WILDCARD} from 'sentry/constants';
+import {Organization, TagCollection} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {FIELD_TAGS, isAggregateField} from 'sentry/utils/discover/fields';
+import {FieldKind} from 'sentry/utils/fields';
+import useApi from 'sentry/utils/useApi';
+import withTags from 'sentry/utils/withTags';
+
+const SEARCH_SPECIAL_CHARS_REGEXP = new RegExp(
+  `^${NEGATION_OPERATOR}|\\${SEARCH_WILDCARD}`,
+  'g'
+);
+
+const REPLAY_TAGS = [
+  'browser.name',
+  'browser.version',
+  'countErrors',
+  'countSegments',
+  'device.brand',
+  'device.family',
+  'device.model',
+  'device.name',
+  'dist',
+  'duration',
+  'id',
+  'os.build',
+  'os.kernel_version',
```

Comment:
```suggestion
  'os.name',
  'os.version',
```

per https://github.com/getsentry/sentry/pull/38343#discussion_r964879868